Documentation/sources/push-notification/manual-handling-and-tracking.md

```diff
@@ -0,0 +1,149 @@
+# Manual display and tracking of push notification
+
+## Overview
+This document provides instructions on understanding notification from Adobe Journey Optimizer and subsequently building, displaying, and monitoring push notifications manually with the AEPMessaging extension.
```

Comment:
How about: "This document provides instructions on handling a notification payload received from Adobe Journey Optimizer and..."